python/tests/integration/completions/test_mai_chat_service.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft. All rights reserved.
```

Comment:
I think `test_mistralai_chat_service` is more clear than `test_mai_chat_service`. What are your thoughts?